packages/std/src/errors/std_error.rs

```diff
@@ -158,6 +148,149 @@ impl StdError {
     }
 }
 
+impl PartialEq<StdError> for StdError {
+    fn eq(&self, rhs: &StdError) -> bool {
+        match self {
+            StdError::GenericErr {
+                msg,
+                #[cfg(feature = "backtraces")]
```

Comment:
Okay, so not even using the .. shortcut to ensure that changes to error trigger compiler errors?

Reply:
Yeah, too easy to forget new fields

Reply:
The risk breaking all the contracts is too high. We have to take the burden on us writing longer code once in order to serve others well.